tests/jobTests.py

```
@@ -0,0 +1,407 @@
+                        "empty":{"filename":"testEmpty.csv","status":"ready","type":"csv_record_validation","submissionLocalId":10,"fileType":1},
+                        "missing_header":{"filename":"testMissingHeader.csv","status":"ready","type":"csv_record_validation","submissionLocalId":11,"fileType":1},
+                        "bad_header":{"filename":"testBadHeader.csv","status":"ready","type":"csv_record_validation","submissionLocalId":12,"fileType":2},
+                        "many":{"filename":"testMany.csv","status":"ready","type":"csv_record_validation","submissionLocalId":11,"fileType":3},
+                        "odd_characters":{"filename":"testOddCharacters.csv","status":"ready","type":"csv_record_validation","submissionLocalId":14,"fileType":2},
+                        "many_bad":{"filename":"testManyBadValues.csv","status":"ready","type":"csv_record_validation","submissionLocalId":11,"fileType":4},
+                        "rules":{"filename":"testRules.csv","status":"ready","type":"csv_record_validation","submissionLocalId":16,"fileType":1}}
+
+            # Upload needed files to S3
+            for key in csvFiles.keys():
+                csvFiles[key]["s3Filename"] = self.uploadFile(csvFiles[key]["filename"],user)
+
+            self.jobIdDict = {}
+
+            sqlStatements = []
+            for key in csvFiles.keys():
+                # Create SQL statement and add to list
+                statement = self.createJobStatement(str(Status.getStatus(csvFiles[key]["status"])), str(Type.getType(csvFiles[key]["type"])), str(submissionIDs[csvFiles[key]["submissionLocalId"]]), csvFiles[key]["s3Filename"], str(csvFiles[key]["fileType"]))
+                jobId = self.jobTracker.runStatement(statement)
+                try:
+                    self.jobIdDict[key] = jobId.fetchone()[0]
+                except InvalidRequestError:
+                    # Problem getting result back, may happen for dependency statements
+                    pass
```

maybe a warning here instead of a silent failure?